src/numerical.jl

```diff
@@ -107,9 +109,11 @@ julia> ModelAnalyzer.summarize(ModelAnalyzer.Numerical.DenseConstraint)
 ```
 """
 struct DenseConstraint <: AbstractNumericalIssue
-    ref::JuMP.ConstraintRef
+    ref::MOI.ConstraintIndex
```

Comment:
Open question whether we should add `{F,S}` in many places. Not sure

Reply:
I am not sure as well, so I was keeping it simpler for now.